test/parallel/test-fs-watchfile.js

```diff
@@ -63,3 +63,25 @@ fs.watchFile(enoentFile, {interval: 0}, common.mustCall(function(curr, prev) {
     fs.unwatchFile(enoentFile);
   }
 }, 2));
+
+// Watch events should callback with a filename
+if (common.isLinux || common.isOSX) {
+  const dir = common.tmpDir + '/watch';
+
+  fs.mkdir(dir, common.mustCall(function(err) {
+    assert(!err);
+
+    fs.watch(dir, common.mustCall(function(eventType, filename) {
+      this._handle.close();
+      common.refreshTmpDir();
```

Comment:
For the test to pass on `Windows` you should remove the `common.refreshTmpDir();`. Don't worry about it, the test harness will take care of it.